test/test-signal.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// Testcase to produce NodeReport on signal
+if (process.argv[2] === 'child') {
+  require('../').setEvents("signal");
+
+  function myLoop() {
+  var list = [];
+    for (var i=0; i<10000000000; i++) {
+      for (var j=0; i<1000; i++) {
+        list.push(new MyRecord());
+      }
+      for (var j=0; i<1000; i++) {
+        list[j].id += 1;
+        list[j].account += 2;
+      }
+      for (var j=0; i<1000; i++) {
+        list.pop();
+      }
+    }
+  }
+
+  function MyRecord() {
+    this.name = 'foo';
+    this.id = 128;
+    this.account = 98454324;
+  }
+
+  myLoop();
+} else {
+  const tap = require('tap');
+  const common = require('./common.js');
+  const fs = require('fs');
+  const spawn = require('child_process').spawn;
+
+  if (process.platform === 'win32') {
+    tap.plan(0, 'Signals not supported on Windows');
```

Comment:
some test parsers fail when there are no asserts, I would change this to a `tap.ok(false, {skip: 'unsupported on windows'), ...` or something of the sort